actionpack/test/controller/request_forgery_protection_test.rb

```diff
@@ -920,15 +934,15 @@ def test_accepts_global_csrf_token
     assert_response :success
   end
 
-  def test_ignores_params
+  def test_chomps_slashes
```

Comment:
I think we should be ignoring params on the token, otherwise the token will depend on the param and we only want it to depend on the path. Are they still being ignored?

Reply:
Yes, we should be.  But that case was testing params in `PATH_INFO` on the request.  But webservers already eliminate parameters from `PATH_INFO` so we don't need to remove them ourselves.

```
[aaron@TC-2 ~]$ cat test.ru
run ->(env) {
  [200, {}, [env['PATH_INFO']]]
}
[aaron@TC-2 ~]$ rackup test.ru &
[aaron@TC-2 ~]$ Puma starting in single mode...
* Version 4.3.1 (ruby 2.7.0-p-1), codename: Mysterious Traveller
* Min threads: 0, max threads: 16
* Environment: development
* Listening on tcp://127.0.0.1:9292
* Listening on tcp://[::1]:9292
Use Ctrl-C to stop
[aaron@TC-2 ~]$ curl 'http://localhost:9292/foo?ahsoeitnahoseit'
::1 - - [10/Jan/2020:11:30:38 -0800] "GET /foo?ahsoeitnahoseit HTTP/1.1" 200 - 0.0004
/foo⏎                                                                                                                                                                                                                 [aaron@TC-2 ~]$ curl 'http://localhost:9292/foo'
::1 - - [10/Jan/2020:11:30:53 -0800] "GET /foo HTTP/1.1" 200 - 0.0004
/foo⏎                                                                                                                                                                                                                 [aaron@TC-2 ~]$
```